### PR TITLE
Message with image

### PR DESCRIPTION
**Pull request by nietongyu**

### CODE REVIEW

File: app/src/main/res/drawable/rounded_button.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item >
+        <shape android:shape="rectangle"  >
+            <corners android:radius="3dip" />
+        </shape>
+    </item>
+</selector>
```
Comment (jiangzhengwenjz) [inline app/src/main/res/drawable/rounded_button.xml]: no newline at end of text file

File: app/src/main/res/layout/activity_drawing_pad.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:id="@+id/canvas"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+
+    <!--        app:layout_constraintBottom_toTopOf="@id/toolbar"-->
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (jiangzhengwenjz) [inline app/src/main/res/layout/activity_drawing_pad.xml]: no newline at end of text file

File: app/src/main/res/layout/layout_message_chatbox.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        android:layout_width="64dp"
+        android:layout_height="48dp"
+        android:layout_gravity="bottom"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:gravity="center"
+        android:text="SEND"
+    android:textSize="14dp" />
+
+</LinearLayout>
```
Comment (jiangzhengwenjz) [inline app/src/main/res/layout/layout_message_chatbox.xml]: no newline at end of text file

File: app/src/main/res/layout/item_message_img_sent.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="8dp">
+
+    <!--    <TextView-->
```
Comment (jiangzhengwenjz) [inline app/src/main/res/layout/item_message_img_sent.xml]: (This is not a change request) You can inject everything you want to comment out into a single `<!--content-->`

Or is it generated by the IDE?

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/item_message_img_sent.xml (hunk 4 above)
Comment (nietongyu) [reply app/src/main/res/layout/item_message_img_sent.xml]: IDE did this, I deleted

**Review by jiangzhengwenjz — APPROVED** [review]

**Pull request merged**